module/config.mjs

```diff
@@ -2478,6 +2475,21 @@ DND5E.healingTypes = {
 };
 preLocalize("healingTypes", { keys: ["label"] });
 
+/* -------------------------------------------- */
+/*  Movement                                    */
+/* -------------------------------------------- */
+
+/**
+ * Types of terrain that can cause difficult terrain.
+ * @enum {string}
+ */
+DND5E.difficultTerrainTypes = {
+  magical: "DND5E.REGIONBEHAVIORS.DIFFICULTTERRAIN.Type.Magical",
```

Comment:
Likelihood of this having to be turned into an enum of objects in the future: estimated 99%

Reply:
I agree, we should turn this into an enum of objects. 
```suggestion
  magical: { label: "DND5E.REGIONBEHAVIORS.DIFFICULTTERRAIN.Type.Magical" },
```